Patterns/English/English-Numbers.yaml

```diff
@@ -110,7 +110,7 @@ CurrencyRegex: !simpleRegex
   def: (((?<=\W|^)-\s*)|(?<=\b))\d+\s*(B|b|m|t|g)(?=\b)
 #Percentage Regex
 NumberWithSuffixPercentage: !nestedRegex
-  def: ({BaseNumbers.NumberReplaceToken})(\s*)(%|per cents|per cent|cents|cent|percentage|percents|percent)
+  def: ({BaseNumbers.NumberReplaceToken})(\s*)(%(?!{BaseNumbers.NumberReplaceToken})|(per cents|per cent|cents|cent|percentage|percents|percent)\b)

```

Comment:
Isn't the lookback going to cause a big performance hit?
Would just a word boundary check solve the issue?

Reply:
I'm afraid adding word boundaries can't solve this issue because the percentage extractor transforms the text source when extract and add special characters like `@`.

Reply:
Not sure I get it, but OK. What about the performance hit of the current solution?